docs/source/serving/openai_compatible_server.md

```diff
@@ -62,6 +62,32 @@ completion = client.chat.completions.create(
 )
 ```
 
+### Extra HTTP Headers
+
+Only `X-Request-Id` HTTP request header is supported for now.
+
+```python
+completion = client.chat.completions.create(
+  model="NousResearch/Meta-Llama-3-8B-Instruct",
+  messages=[
+    {"role": "user", "content": "Classify this sentiment: vLLM is wonderful!"}
+  ],
+  extra_headers={
+    "x-request-id": "sentiment-classification-00001",
```

Comment:
Just to be sure: does the capitalization of `X-Request-Id` matter?

Reply:
It doesn't; `openai-python` library uses `httpx` for its HTTP client SDK, and `httpx` handles the capitalization part.